makevsix.js

```diff
@@ -0,0 +1,74 @@
+// 
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+require('shelljs/global');
+
+var path = require('path'),
+    fs = require('fs');
+
+var argv = require('yargs')
+    .usage('Usage: $0 <command>')
+    .command('create', 'Create VSIX')
+    .command('publishtest', 'Publish test VSIX')
+    .demand(1)
+    .argv;
+	
+
+var command = argv._[0];
+
+if (! exec("npm --version", {silent: true})) {
+    echo("npm not found. please install npm and run again.")
+    exit(1);
+}
+
+if (exec('tfx --version', {silent: true}).code != 0)
+    echoAndExec("tfx-cli not found. installing", "npm install -g tfx-cli");
+
+installTasks()
+
+if (command === 'create')
+    echoAndExec('Creating VSIX', 'tfx extension create --manifest-globs mobiledevopscordovaextension.json --override {\\\"public\\\":true}');
+else if (command === 'publishtest') {
+    var token = env['PUBLISH_ACCESSTOKEN'];
+    if (token === undefined) {
+        echo("Must set PUBLISH_ACCESSTOKEN environment variable to publish a test VSIX");
+        exit(1);
+    }
+    
+    echoAndExec('Creating and publishing test VSIX...',
+        'tfx extension publish --manifest-globs mobiledevopscordovaextension.json --override { \\\"public\\\": false, \\\"name\\\": \\\"Cordova Build-Dev\\\", \\\"id\\\": \\\"cordova-extension-dev\\\", \\\"publisher\\\": \\\"ms-mobiledevops-test\\\"} --share-with mobiledevops x04ty29er --token ' + env['PUBLISH_ACCESSTOKEN']);
```

Comment:
We should probably put this in a var up top called something like devManifestOverrides and prodManifestOverrides or something and append this as a json string here.